backend/organization/utility/organization.py

```diff
@@ -138,3 +140,40 @@ def add_organization_member(organization, user, user_role, role_in_organization)
     )
 
     return
+
+
+def check_edit_exisiting_name(organization, name): 
+    # allows an org to change capitilization of their name
+    if (Organization.objects.filter(name__iexact=name).exclude(id=organization.id).exists()):
+        return True
+
+    if (OrganizationTranslation.objects.filter(
+                name_translation__iexact=name
+            ).exclude(organization=organization)):
+        return True
+
+    return False
+
+
+def check_create_existing_name(name):
+    if check_existing_name(name) or check_existing_name_translation(name):
+        return True
+    return False
+
+
+def check_existing_name(name):
+    if Organization.objects.filter(name__iexact=name).exists():
+            return True
+    return False
+
+
+def check_existing_name_translation(name):
+    if OrganizationTranslation.objects.filter(
+            name_translation__iexact=name
+        ).exists():
+            return True
+    return False
+
+
+def get_existing_name_message(name):
+    return _("The name {} is already in use. Please use another name.").format(name)
```

Comment:
Here instead we should display something along these lines in the frontend "Someone has already created the organization <a href={link_to_org} target="_blank">{orgname}</a>. Please join the organization or use a different name. If you're having problems please contact support@climateconnect.earth.". That's a pretty long message but I think in the rare case that somebody tries to use an organization name that already exists we need to let them know that they can join the existing org if that was the one they were trying to create. Also we should display them a contact option incase somebody else created their organization without their permission.

Reply:
Made the change addressing this, however, I wasn't able to make the email a mailto link.